cmds/wifi/server.go

```diff
@@ -254,6 +255,7 @@ func (ws WifiServer) buildRouter() *mux.Router {
 	r.HandleFunc("/", ws.displayStateHandle).Methods("GET")
 	r.HandleFunc("/refresh", ws.refreshHandle).Methods("POST")
 	r.HandleFunc("/connect", ws.connectHandle).Methods("POST")
+	r.PathPrefix("/css/").Handler(http.StripPrefix("/css/", http.FileServer(http.Dir("/etc/sos/html/css"))))
```

Comment:
What happens if the file doesn't exist?

Reply:
It just won't load the css. The html will persist though and you can still navigate the different services

Reply:
that's good. This is good to go